client/src/pages/teds/charts/PrctIpccPublicationsForFiveCountries.tsx

```diff
@@ -0,0 +1,219 @@
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations (WG2)",
+  };
+  const filter_wg2cross = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[2] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations - risques locaux (WG2 cross)",
+  };
+  const filter_wg3 = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[3] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+    },
+    name: "Atténuations (WG3)",
+  };
+  const filter_wg1_and_wg2 = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Sciences Physique et Adaptations",
+  };
+
+  const filter_wg2_and_wg2cross = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations et Adaptations - risques locaux",
+  };
+  const filter_wg2_and_wg3 = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+    },
+    name: "Adaptation et Atténuation",
+  };
+
+  const filters = [
+    filter_wg1,
+    filter_wg2,
+    filter_wg2cross,
+    filter_wg3,
+    filter_wg1_and_wg2,
+    filter_wg2_and_wg2cross,
+    filter_wg2_and_wg3,
+  ];
+
+  const countries = [
+    "United States",
+    "United Kingdom",
+    "Germany",
+    "France",
+    "China",
+  ];
+
+  const responses = [
+    useQueryResponse(filters[0].body, 50, filters[0].name),
+    useQueryResponse(filters[1].body, 50, filters[1].name),
+    useQueryResponse(filters[2].body, 50, filters[2].name),
+    useQueryResponse(filters[3].body, 50, filters[3].name),
+    useQueryResponse(filters[4].body, 50, filters[4].name),
+    useQueryResponse(filters[5].body, 50, filters[5].name),
+    useQueryResponse(filters[6].body, 50, filters[6].name),
+  ];
+
+  const response_total = useQueryResponse(
+    filter_ipcc.body,
+    50,
+    filter_ipcc.name
+  );
+
+  const loadings = responses.map((response) => response.isLoading);
+  const isLoad = (currentValue) => currentValue === true;
```

Comment:
why do you need this ?

Reply:
for aggregations, i will do it on elastic not on OpenAlex. 
loadings is an array with all the loadings of the useQueryResponse responses, I check that all the responses are loaded otherwise some data is not loaded and I can't manipulate it.

Reply:
Yes, but as `isLoading` is a boolean (https://tanstack.com/query/v4/docs/framework/react/reference/useQuery), you just need to write something like :
```
if (responses.map((response) => response.isLoading).some((item) => item)) {
    return <div>Loading...</div>;
 }
```

Reply:
aaah ok